crates/rust-macro/src/lib.rs

```diff
@@ -140,14 +152,31 @@ impl Config {
 mod kw {
     syn::custom_keyword!(std_feature);
     syn::custom_keyword!(raw_strings);
-    syn::custom_keyword!(macro_export);
-    syn::custom_keyword!(macro_call_prefix);
-    syn::custom_keyword!(export_macro_name);
     syn::custom_keyword!(skip);
     syn::custom_keyword!(world);
     syn::custom_keyword!(path);
     syn::custom_keyword!(inline);
     syn::custom_keyword!(ownership);
+    syn::custom_keyword!(world_exports);
+    syn::custom_keyword!(interface_exports);
+    syn::custom_keyword!(resource_exports);
+    syn::custom_keyword!(stubs);
+    syn::custom_keyword!(export_prefix);
+}
+
+#[derive(Clone)]
+struct Export {
+    key: syn::LitStr,
+    value: syn::Ident,
```

Comment:
Could this be `syn::Path` so that the implementing type names for the resources don't have to be in the current scope of where bindings are generated?

Reply:
Done.